cpp/include/cudf/strings/detail/converters.hpp

```diff
@@ -107,6 +107,29 @@ std::unique_ptr<column> from_timestamps(column_view const& timestamps,
                                         cudaStream_t stream,
                                         rmm::mr::device_memory_resource* mr);
 
+/**
+ * @copydoc to_durations(strings_column_view const&,data_type,std::string
```

Comment:
I don't know how copydoc works. so I'm curious why `@param stream` is required, when it seems like the others are not?

Reply:
These are detail APIs. public APIs does not have `stream`. We copy the doc from public APIs to detail APIs and add extra argument `@param stream`.